src/Indexer.js

```diff
@@ -69,6 +69,20 @@ export default class Indexer {
     })
   }
 
+  // https://www.elastic.co/guide/en/elasticsearch/reference/current/analysis-asciifolding-tokenfilter.html
+  indexSettings() {
+    return {
+      analysis : {
+        analyzer : {
+          default : {
+            tokenizer : 'standard',
+            filter : ['asciifolding']
+          }
+        }
+      }
+    }
+  }
```

Comment:
one other thing on this configuration:  i'd considered putting this index settings stuff in the environment configs... but the tests rely on things being set up as they are in the code here, so in the end, it felt more appropriate to put that info in the code.  totally open to argument/suggestion on that.  but it didn't seem like we'd be overriding the index analysis settings.  and it did seem like something we want to test the behavioral effect of.